internal/provider/resource_value_test.go

```diff
@@ -50,10 +50,126 @@ func TestAccResourceValue(t *testing.T) {
 	})
 }
 
+func TestAccResourceValueWithSecretValue(t *testing.T) {
+	appID := fmt.Sprintf("val-test-app-%d", time.Now().UnixNano())
+	key := "VAL_SECRET_1"
+	orgID := os.Getenv("HUMANITEC_ORG_ID")
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { testAccPreCheck(t) },
+		ProtoV6ProviderFactories: testAccProtoV6ProviderFactories,
+		Steps: []resource.TestStep{
+			// Create and Read testing
+			{
+				Config: testAccResourceVALUETestAccResourceValueSecret(appID, key, "Example value with secret"),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					resource.TestCheckResourceAttr("humanitec_value.app_val_with_secret", "key", key),
+					resource.TestCheckResourceAttr("humanitec_value.app_val_with_secret", "description", "Example value with secret"),
+					resource.TestCheckResourceAttr("humanitec_value.app_val_with_secret", "secret_ref.ref", fmt.Sprintf("orgs/%s/apps/%s/secret_values/%s/.value", orgID, appID, key)),
+				),
+			},
+			// ImportState testing
+			{
+				ResourceName: "humanitec_value.app_val_with_secret",
+				ImportState:  true,
+				ImportStateIdFunc: func(s *terraform.State) (string, error) {
+					return fmt.Sprintf("%s/%s", appID, key), nil
+				},
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"secret_ref", "value"},
```

Comment:
I had to ignore `secret_ref` too, _maybe_ because we are not importing this in `ImportState`. I'm not sure this is the right thing to do tbh.